### DISCUSSION TITLE
kubectl: cannot execute binary file: exec format error

### DISCUSSION THREAD

**Discussion by zakir-hush** [ish-app/ish#1873] [Q&A]
Hi,   Am new to linux and infrastructure side.. I have installed iSH in my ipad and trying to run kubectl. I have Installed kubectl binary with curl but cant execute <kubectl> it gives me error "cannot execute binary file exec format error". 
wondering if it is this tested? can I use kubectl in iSH ?   https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux 

Cheers,
Zak.

**Comment by itskenny0** [comment]
Change ‘amd64’ in the download cURL command to ‘i386’.

edit: nvm, they don’t seem to provide a non-64bit binary.

**Reply by zakir-hush** [reply]
Thanks for your quick response Kenny, you are right changing to i386 couldn't download the file. Hopefully there will be a solution soon. 

cheers!
